### PR TITLE
Allow for origin and location specific contact information in emails …

### LINKED ISSUE #224: Contact info for mediated page confirmation emails
Origin =

SPEC-COLL
(650) 725-1022
specialcollections@stanford.edu

HOOVER
(650) 723-2058
hoovercirc@stanford.edu

HV-ARCHIVE
(650) 723-3563
archives@hoover.stanford.edu

HOPKINS
(831) 655-6229
HMS-Library@lists.stanford.edu

Location =

PAGE-MP
(650) 723-2746
brannerlibrary@stanford.edu

### PR DESCRIPTION
**Pull request by jkeck**
…with a default fallback.

Closes #224

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -53,9 +53,24 @@ en:
       scans: 'Scan requests'
   confirmation_email:
     contact:
-      default: "  (650) 723-1493\n  greencirc@stanford.edu"
-    default:
-      from: 'greencirc@stanford.edu'
+      HOOVER:
```
Comment (cbeer) [inline config/locales/en.yml]: Do you think we should promote these into configuration? Without any of the formatting, i18n starts to feel suspect to me..

File: config/locales/en.yml [hunk 2]
```diff
@@ -53,9 +53,24 @@ en:
...
+        phone: '(650) 723-2058'
+        email: 'hoovercirc@stanford.edu'
+      HV-ARVHIVE:
```
Comment (cbeer) [inline config/locales/en.yml]: `ARCHIVE`?

File: config/locales/en.yml (hunk 1 above)
Comment (jkeck) [inline config/locales/en.yml]: I could go either way.  I'm using I18n's default value behavior to fallback but that could easily be done with a hash as well.  I'll migrate them into `application.rb` and push it up momentarily.

File: config/locales/en.yml (hunk 2 above)
Comment (jkeck) [inline config/locales/en.yml]: Whoops!

File: config/locales/en.yml (hunk 1 above)
Comment (jkeck) [inline config/locales/en.yml]: :+1: take a look now.

**Pull request merged**